Java/src/com/couchbase/litecore/C4Database.java

```diff
@@ -81,6 +81,14 @@ public long nextDocExpiration() {
         return nextDocExpiration(handle);
     }
 
+    public int purgeExpiredDocs() {
+        return purgeExpiredDocs(handle);
+    }
+
+    public boolean purgeDoc(String docID) throws LiteCoreException {
```

Comment:
The returned type should be void not boolean. Java already throws LiteCoreException when failure occurs so there is no need to return boolean.